source/linux/Foundational_Components/U-Boot/UG-QSPI.rst

```diff
@@ -279,83 +516,124 @@ documented in U-boot SPI section.
-                 +----------------------------+
+             0x0 +---------------------+
+                 |   ospi.tiboot3      |
+                 |   (1m)              |
+        0x100000 +---------------------+
+                 |   ospi.tispl        |
+                 |   (2m)              |
+        0x300000 +---------------------+
+                 |   ospi.u-boot       |
+                 |   (4m)              |
+        0x700000 +---------------------+
+                 |   ospi.env          |
+                 |   (256k)            |
+        0x740000 +---------------------+
+                 |   ospi.env.backup   |
+                 |   (256k)            |
+        0x780000 +---------------------+
+                 |   padding           |
+                 |   (512k)            |
+        0x800000 +---------------------+
+                 |   ospi.rootfs       |
+                 |   (55m)(ubifs)      |
+       0x3FC0000 +---------------------+
+                 |   ospi.phypattern   |
+                 |   (256k)            |
+                 +---------------------+
 
 .. ifconfig:: CONFIG_part_variant in ('AM62X')
 
-    AM62x Starter Kit (SK) has a Cypress S28HS512T flash and sysfw is bundled with tiboot3.bin.
+    **AM62x SK**
+
+    AM62x Starter Kit (SK) has a Cypress S28HS512T OSPI NOR flash and SYSFW is
+    bundled with tiboot3.bin.
 
-    Below are three methods which can be used to flash the OSPI device. For all methods, we will load the
-    bootloders into memory and then flash each to OSPI at the respective addresses.
+    Below are two methods which can be used to flash the OSPI NOR device. For
+    all methods, we will load the bootloaders into memory and then flash each
+    to OSPI NOR at the respective addresses.
 
-    **Flashing images to OSPI using TFTP server**
+    **Flashing Images to OSPI NOR using TFTP server**
 
-    In this example, use tftp-hpa in Ubuntu as tftp server. Assume bootloader names are 'tiboot3.bin', 'tispl.bin',
-    'u-boot.img'. Verify ethernet connection between AM62x EVM and host machine before proceeding.
+    In this example, use tftp-hpa in Ubuntu as tftp server. Assume bootloader
+    names are 'tiboot3.bin', 'tispl.bin', 'u-boot.img'. Verify ethernet
+    connection between AM62x SK and host machine before proceeding.
 
     1. Setup TFTP server in Host machine
 
     .. code-block:: console
 
       # For complete instructions refer to: https://help.ubuntu.com/community/TFTP
       $ sudo apt install tftp-hpa
-      $ sudo vi /etc/default/tftpd-hpa #optional to change tftp directory and other options
-      $ sudo chown -R tftp /tftp #change owner/group of new directory /tftp
-      $ sudo systemctl restart tftpd-hpa #restart server
+      $ sudo vi /etc/default/tftpd-hpa # optional to change tftp directory and other options
+      $ sudo chown -R tftp /tftp # change owner/group of new directory /tftp
+      $ sudo systemctl restart tftpd-hpa # restart server
 
```

Comment:
This section is common between NAND and NOR. May be worth making it a prerequisite section and having both ifdef branches link back to it.

Reply:
I'm trying to use ' .. include:: ' for this. Facing some issues. Will post a separate PR.

Rest of the suggested changes are made.